genienlp/arguments.py

```diff
@@ -401,12 +401,8 @@ def parse_argv(parser):
 
     # NED args
     parser.add_argument('--do_ned', action='store_true', help='Collect and use entity features during training')
-    parser.add_argument(
-        '--min_entity_len', type=int, default=1, help='Minimum length for entities when ngrams database_lookup_method is used '
-    )
-    parser.add_argument(
-        '--max_entity_len', type=int, default=4, help='Maximum length for entities when ngrams database_lookup_method is used '
-    )
+    parser.add_argument('--min_entity_len', type=int, default=1, help='Minimum token-length of entities in ngram-based lookup')
+    parser.add_argument('--max_entity_len', type=int, default=4, help='Maximum token-length of entities in ngram-based lookup')
```

Comment:
I am like 50% confident that I saw this argument in Bootleg code as well, so why this help text talks about ngram-based lookup?

Reply:
Yes. This argument is shared between the two scripts. Here it's used for the naive NED approach which does NER right before training (as opposed to bootleg which does it in a separate step by running run_bootleg.py)

Reply:
This is confusing. How about "Minimum token-length of entities (does not apply to Bootleg)"?